shell/common/rasterizer.cc

```diff
@@ -469,7 +469,11 @@ RasterStatus Rasterizer::DrawToSurface(flutter::LayerTree& layer_tree) {
         raster_status == RasterStatus::kSkipAndRetry) {
       return raster_status;
     }
-    if (external_view_embedder != nullptr) {
+    if (external_view_embedder != nullptr &&
+        (!raster_thread_merger_ || raster_thread_merger_->IsMerged())) {
```

Comment:
Wait, shouldn't this be `raster_thread_merger_ && raster_thread_merger_->IsMerged()`?

Reply:
Some platforms don't support `raster_thread_merger_`, I assume those platforms run `external_view_embedder->SubmitFrame` instead of `frame->Submit` on raster thread. 

So the check says only check if threads are merged when raster_thread_merger_ is supported

Reply:
Ok, for iOS that isn't a problem at least since raster_thread_merge_ will be set before SubmitFrame ever happens.